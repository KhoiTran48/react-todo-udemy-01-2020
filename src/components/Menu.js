import {NavLink} from 'react-router-dom'
import React, { Component } from 'react'

export default class Menu extends Component {
    render() {
        return (
            <div>
                <NavLink to="/" exact>Dashboard</NavLink>
                <NavLink to="/add">Add Task</NavLink>
            </div>
        )
    }
}
