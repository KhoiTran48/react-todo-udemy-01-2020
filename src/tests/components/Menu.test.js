import React from 'react'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import Menu from '../../components/Menu'

test("render Menu correctly", ()=>{
    const redenerer = new ReactShallowRenderer();
    redenerer.render(<Menu/>)
    // console.log(redenerer.getRenderOutput());
    expect(redenerer.getRenderOutput()).toMatchSnapshot();
})