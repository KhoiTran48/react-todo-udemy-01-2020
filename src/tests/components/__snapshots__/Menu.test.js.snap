// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Menu correctly 1`] = `
<div>
  <NavLink
    activeClassName="active"
    aria-current="page"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="active"
    aria-current="page"
    to="/add"
  >
    Add Task
  </NavLink>
</div>
`;
